Project.App/Project.Api/Models/Games/BlackjackConfig.cs

```diff
@@ -8,4 +8,5 @@ public record BlackjackConfig : GameConfig
     public long StartingBalance { get; set; } = 1000;
     public long MinBet { get; set; } = 0;
     public TimeSpan BettingTimeLimit { get; set; } = TimeSpan.FromSeconds(60);
+    public TimeSpan TurnTimeLimit { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
Note that `TurnTimeLimit` is already defined in the parent record `GameConfig`. If you're trying to set a new default, the `override` keyword should be used.

```suggestion
    public override TimeSpan TurnTimeLimit { get; set; } = TimeSpan.FromSeconds(30);
```